### ISSUE TITLE
Blacklight: fix CSS bugs

### ISSUE THREAD

**Issue by mikeapp** [yalelibrary/YUL-DC#2056]
**Story**

Sonarcloud has identified a number of potential issues with CSS in yul-dc-blacklight.  These are primarily related to fonts.   https://sonarcloud.io/project/issues?resolved=false&types=BUG&id=yalelibrary_yul-dc-blacklight

HTML  bugs will be addressed in another issue.

**Acceptance**
- [x] Fix CSS bugs

**Comment by martinlovell** [comment]
Running into some issues/questions.

There are a few places where we had the "font" property in CSS without a size, so they weren't applied.  If I fix those properties to be "font-family" the fonts are applied.  I'm not sure if we want this in all cases or what the original intent was.

I can go through an correct the CSS, but I think we need to have everything reviewed to verify that we really want the font families applied.

Here's an example in advanced search. (Notice the H1 becomes Italic and the select boxes font family changes slightly:
![font-family-affect.gif](https://images.zenhubusercontent.com/5e5e9bbd38fda217bdabbde0/ca044d0c-9a2c-43a5-89ab-c2e3ac265cb3)

**Comment by mikeapp** [comment]
Whoa, that's a change.  It might be helpful to dig up Matt's design if we can still find it.  In any case we can keep this in a PR and maybe look at it on demo once the UV4 work is done.

**Comment by martinlovell** [comment]
Draft PR: https://github.com/yalelibrary/yul-dc-blacklight/pull/794

**Comment by martinlovell** [comment]
I could update the PR to remove all the "font" entries in CSS that were invalid instead of convert to "font-family".
This would keep everything as is.  We can add new font-family tags if we decide they were supposed to be applied.

**Comment by martinlovell** [comment]
The PR was updated to remove all the "font" css entries that had been ignored so there should be no visual changes for the user.

**Comment by martinlovell** [comment]
v2.59.66

**Comment by martinlovell** [comment]
These aren't visible to the user.  Everything should look the same in Blacklight.
